docs/docs/guides/create-job.md

```diff
@@ -107,8 +126,18 @@ labels:
    orchestrator: optimus
 dependencies: []
 hooks: []
+metadata:
```

Comment:
Can you also add this with 1-2 description on [this](https://odpf.github.io/optimus/docs/concepts/overview#job-specification) section?

Reply:
sure, updating

Reply:
it's updated